sale_order_type/views/account_move_views.xml

```diff
@@ -7,7 +7,7 @@
             <field name="journal_id" position="before">
                 <field
                     name="sale_type_id"
-                    attrs="{'invisible': [('type', 'not in', ['in_invoice', 'in_refund', 'out_invoice', 'out_refund'])]}"
```

Comment:
I wonder which was the intention of the original contributor including explicitly the vendor bills.

Reply:
I was wondering the same

Reply:
It's a mistake :sweat_smile: 
this change is correct :+1: